contrib/adsk/libraries/adsklib/adsklib_defs.mtlx

```diff
@@ -144,15 +144,15 @@
     <input name="surface_cutout" type="color3" value="1, 1, 1" />
     <input name="surface_albedo" type="color3" value="1, 1, 1" />
     <input name="opaque_albedo" type="color3" value="1, 1, 1" />
-    <input name="surface_roughness" type="float" value="0" />
-    <input name="surface_anisotropy" type="float" value="0" />
-    <input name="surface_rotation" type="float" value="0" />
+    <input name="surface_roughness" type="float" value="0" uimin="0.0" uimax="1.0" />
+    <input name="surface_anisotropy" type="float" value="0" uimin="0.0" uimax="1.0" />
+    <input name="surface_rotation" type="float" value="0" uimin="0.0" uimax="180.0" />
     <input name="opaque_f0" type="float" value="1" />
     <input name="opaque_translucency" type="boolean" value="false" />
     <input name="opaque_mfp_modifier" type="color3" value="1, 1, 1" />
-    <input name="opaque_mfp" type="float" value="1" unittype="distance" />
+    <input name="opaque_mfp" type="float" value="1" unittype="distance" uimin="0.0" uisoftmax="100.0" />
     <input name="opaque_emission" type="boolean" value="false" />
-    <input name="opaque_luminance" type="float" value="1.0" />
+    <input name="opaque_luminance" type="float" value="1.0" uimin="0.0" uisoftmax="1500.0" />
```

Comment:
can we explicit about the physical units used.
`cd/m^2` if it is already available in the registered units